tasks/sync-granule/GranuleFetcher.js

```diff
@@ -123,7 +130,7 @@ class GranuleFetcher {
     );
 
     const downloadFiles = filesWithChecksums
-      .filter((f) => !isChecksumFile(f))
+      .filter((f) => this.syncChecksumFiles || !isChecksumFile(f))
```

Comment:
If you do make `syncChecksumFiles` a parameter, it probably wouldn't be a bad idea to update this method to take a `params={}` object rather than the positional parameters that it's taking now. This isn't part of a published API, so you can make that change without breaking anyone.